packages/tasit-action/src/testHelpers/mochaSetup.js

```diff
@@ -0,0 +1,29 @@
+// Chai
+import chai, { expect } from "chai";
+global.expect = expect;
+chai.use(require("chai-as-promised"));
+
+// Sinon
+import sinon from "sinon";
+global.sinon = sinon;
+
+// Helpers
+import {
+  mineBlocks,
+  createSnapshot,
+  revertFromSnapshot,
+  wait,
+  waitForEthersEvent,
+} from "./helpers";
+global.mineBlocks = mineBlocks;
+global.createSnapshot = createSnapshot;
+global.revertFromSnapshot = revertFromSnapshot;
+global.wait = wait;
+global.waitForEthersEvent = waitForEthersEvent;
+
+// External helpers
+//
+// Note:  Using dist file because babel doesn't compile node_modules files.
+// Any changes on src should be followed by compilation to avoid unexpected behaviors.
```

Comment:
`lerna bootstrap` runs `prepare` in all bootstrapped packages.
Refs: https://github.com/lerna/lerna/tree/master/commands/bootstrap

Reply:
Ah ok, that explains it!

Reply:
```suggestion
// Any changes on src should be followed by compilation to avoid unexpected behaviors.
// Note that lerna bootstrap does this for you since it
// runs prepare in all bootstrapped packages.
// Refs: https://github.com/lerna/lerna/tree/master/commands/bootstrap
```